docs/0.1/using_grid_features.md

```diff
@@ -0,0 +1,43 @@
+# Using Grid Features
+
+<!--
+  Copyright (c) 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+Most Grid features are designed to interact with Pike organizations and Schema
+definitions.
+
+For example, each product requires an owning organization, one or more agents
+that have permission to create and update the product, and a product schema
+that defines the structure and format of product properties.
+Likewise, each product schema must have an owning organization and an agent
+(or agents) with appropriate permissions.
+
+Follow these general steps when creating a new item such as a Grid Product.
+
+1. Create an organization with at least one agent that has the permissions to
```

Comment:
The following steps refer to the item as `Product` might change it to product here as well for consistency